docs/ops.html

```diff
@@ -1249,7 +1249,7 @@ <h4 class="anchor-heading"><a id="prodconfig" class="anchor-link"></a><a href="#
 
   <h3 class="anchor-heading"><a id="java" class="anchor-link"></a><a href="#java">6.6 Java Version</a></h3>
 
-  Java 8, Java 11, Java 17, and Java 21 are supported.
+  Java 8, Java 11, Java 17, Java 21, and Java 23 are supported.
```

Comment:
Was this change intentional?

Reply:
Oh! It should not have been sent actually. Thanks! I will remove it.